modules/installation-creating-gcp-bootstrap.adoc

```diff
@@ -32,13 +36,38 @@ have to contact Red Hat support with your installation logs.
 section of this topic and save it as `04_bootstrap.py` on your computer. This
 template describes the bootstrap machine that your cluster requires.
 
-. Export the following variables required by the resource definition:
+. Export the variables that the deployment template uses:
+.. Export the control plane subnet location:
 +
+ifndef::shared-vpc[]
 ----
 $ export CONTROL_SUBNET=`gcloud compute networks subnets describe ${INFRA_ID}-master-subnet --region=${REGION} --format json | jq -r .selfLink`
```

Comment:
Need to specify --project --account:

    export CONTROL_SUBNET=$(gcloud compute networks subnets describe ${HOST_PROJECT_CONTROL_SUBNET} --region=${REGION} --project ${HOST_PROJECT} --account ${HOST_PROJECT_ACCOUNT} --format json | jq -r .selfLink)